docs/sphinx/formats/perkinelmer-columbus.rst

```diff
@@ -0,0 +1,47 @@
+.. index:: PerkinElmer Columbus
+.. index:: .xml, .csv, .tif
+
+PerkinElmer Columbus
+===============================================================================
+
+Extensions: .xml, .csv, .tif
+
+
+Owner: `PerkinElmer <http://www.perkinelmer.com/>`_
+
+**Support**
+
+
+BSD-licensed: |no|
+
+Export: |no|
+
+Officially Supported Versions: 
+
+Reader: ColumbusReader (:bfreader:`Source Code <ColumbusReader.java>`, :doc:`Supported Metadata Fields </metadata/ColumbusReader>`)
+
+
+
+
+We currently have:
+
+* a few example datasets
+
+We would like to have:
+
```

Comment:
Do we have an official specification? If not it might be good to list under 'We would like to have'